src/core/server/status/status_service.test.ts

```diff
@@ -321,6 +321,160 @@ describe('StatusService', () => {
       });
     });
 
+    describe('coreOverall$', () => {
+      it('exposes an overall summary of core services', async () => {
+        const setup = await service.setup(
+          setupDeps({
+            elasticsearch: {
+              status$: of(degraded),
+            },
+            savedObjects: {
+              status$: of(degraded),
+            },
+          })
+        );
+        expect(await setup.coreOverall$.pipe(first()).toPromise()).toMatchObject({
+          level: ServiceStatusLevels.degraded,
+          summary: '[2] services are degraded',
```

Comment:
Yea, that seems like a nice improvement. It may impact quite a few tests though, as we'll want to do the same for the global `overall` status, so I'll do it as a follow-up as soon as this one is merged.